docs/project_structure.md

```diff
@@ -40,18 +40,27 @@ The `nginx` directory contains a NGINX configuration file and Docker container.
 
 The `tools` directory contains some shell scripts. They are used for CI, CD and so on.
 
+Also, there are directories and files contain doccano v0.x codes.
+In the future, they will be integrated into the currect code or removed:
+
+```
+/
+├── app/
+├── └── server/
+└── docker-compose.prod.yml
+```
```

Comment:
Which files could we add here?

Reply:
Files under `app` directory is related to doccano v0.x and v1.x. In particular, `server` directory is related to v0.x.

```
.
├── api/
├── app/
├── authentification/
├── server/
├── tools/
├── Dockerfile
├── manage.py
└── requirements.txt
```

Reply:
This paragraph is about v0.x only, look, please.

* /app/server/
* /docker-compose.prod.yml

I missed something?

Reply:
`/docker-compose.prod.yml` is for v1.x. Change it to `/docker-compose.yml`.

Reply:
Ooough..... I'm sorry. Thanks! Done.

Reply:
@Hironsan , BTW I didn't know `app/server` is used in v1.x _default_ (Docker Compose) installation, too. But now [I see](https://github.com/doccano/doccano/blob/master/app/server/management/commands/create_admin.py).

Should I change something in this paragraph again, what do you think?